master/buildbot/steps/source/git.py

```diff
@@ -387,6 +387,8 @@ def _fetch(self, _):
             if rc == RC_SUCCESS:
                 fetch_required = False
 
+        abandonOnFailure = not self.retryFetch and not self.clobberOnFailure
```

Comment:
@sa2ajj yes. I just used whatever was already in the code in order to be consistent.